cmd/cosign/cli/clean.go

```diff
@@ -49,9 +50,21 @@ func Clean() *cobra.Command {
 	return cmd
 }
 
+// confirmPromptDestructive prompts the user for confirmation for an action. Ignores
+// skipConfirmation.
+func confirmPromptDestructive(msg string) (bool, error) {
```

Comment:
@haydentherapper do you mean replacing this with a `cosign.ConfirmPrompt(msg, false)`? If so that sounds like an easy fix I can send out shortly.

Reply:
Yea, just having one destructive prompt function, one normal confirm prompt function.